app/javascript/packages/document-capture/components/acuant-selfie-capture-canvas.jsx

```diff
@@ -17,13 +17,20 @@ function FullScreenLoadingSpinner({ fullScreenRef, onRequestClose, fullScreenLab
   );
 }
 
-function AcuantSelfieCaptureCanvas({ fullScreenRef, onRequestClose, fullScreenLabel }) {
+function AcuantSelfieCaptureCanvas({
+  fullScreenRef,
+  onRequestClose,
+  fullScreenLabel,
+  imageCaptureText,
+}) {
   const { isReady } = useContext(AcuantContext);
   // The Acuant SDK script AcuantPassiveLiveness attaches to whatever element has
   // this id. It then uses that element as the root for the full screen selfie capture
   const acuantCaptureContainerId = 'acuant-face-capture-container';
   return isReady ? (
```

Comment:
```suggestion
    // The <p> to display the text over the selfie capture can be moved to another component
    // without too much work, putting it here to keep it conceptually grouped with the selfie capture canvas 
    return isReady ? (
```

Reply:
Either way, personally think it's too much for an additional component just for 3 lines of static code, actually just one line.

Reply:
Hm, interesting. Seems like that comment might be more confusing than it's worth. Agreed that the `<p>` should't be it's own component.

What I meant was that it actually doesn't matter where the `<p>` is (as long as it's in the selfie components) because the CSS removes it from the document flow.